resources/user-agents/browsers/waterfox/waterfox-51-on.json

```diff
@@ -41,7 +41,8 @@
           "match": "Mozilla/5.0 (#PLATFORM#) Gecko* Firefox/* Waterfox/#MAJORVER#.#MINORVER#*",
           "device": "Macintosh",
           "platforms": [
-            "OSX_10_12", "OSX_10_11", "OSX_10_10", "OSX_10_9", "OSX_10_8", "OSX_10_7", "OSX_10_6", "OSX_10_5",
+            "OSX_10_13", "OSX_10_12", "OSX_10_11", "OSX_10_10",
+            "OSX_10_9", "OSX_10_8", "OSX_10_7", "OSX_10_6", "OSX_10_5",
```

Comment:
Are these old versions really supported until now?

Reply:
According to their site, they still support OSX 10.5+:

https://www.waterfoxproject.org/downloads